gammapy/background/off_data_background_maker.py

```diff
@@ -203,3 +224,81 @@ def save_models(self, modeltype):
         """
         for ngroup in range(self.ntot_group):
             self.save_model(modeltype, ngroup)
+
+    def make_bkg_model_link(self, data_store, modeltype, out_dir_background_model=None, filename_obs_group_table=None):
+        """
```

Comment:
Maybe rename this method to `make_bkg_index_table`?

And please add a one-line description to the docstring at the top:
```
"""Make background model index table.

...
"""